packages/ogm/src/generate.test.ts

```diff
@@ -51,16 +51,13 @@ describe("generate", () => {
         expect(generated).toMatchInlineSnapshot(`
             "import { SelectionSetNode, DocumentNode } from \\"graphql\\";
             export type Maybe<T> = T | null;
-            export type InputMaybe<T> = Maybe<T>;
```

Comment:
It looks like this is reverting a change in #713 here:
https://github.com/neo4j/graphql/pull/713/files#diff-7f1bf5c3db524a84130ef78d588a180f48e311278a2243315bb523df59f80e56